src/main/java/org/digitalstack/chess/Pawn.java

```diff
@@ -44,7 +46,19 @@ private void setPieceColor(PieceColor value) {
     }
 
     public void move(MovementType movementType, int newX, int newY) {
-        throw new UnsupportedOperationException("Need to implement Pawn.move()") ;
+    	if(MovementType.MOVE.equals(movementType)) {
+    		if(this.getYCoordinate() > 1 && (this.getXCoordinate() == newX || this.getXCoordinate()-1 == newX)) {
+    			this.setXCoordinate(newX);
+                this.setYCoordinate(newY);
+               }
+    	   }
+    	else {
```

Comment:
Replace the else with an if for MovementType.CAPTURE, it's clearer what the code is doing this way.
Maybe use a switch statement.